tasks/jsdoc/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
I will remove this package if we decide to merge eventually, it is just for testing the functionality